pallets/circuit/src/lib.rs

```diff
@@ -552,12 +553,14 @@ pub mod pallet {
         pub fn on_extrinsic_trigger(
             origin: OriginFor<T>,
             side_effects: Vec<SideEffect<T::AccountId, BalanceOf<T>>>,
-            _sequential: bool,
+            speed_mode: SpeedMode,
```

Comment:
will there always be a speed mode? If we made this an option we can pass None and it will use a sensible default.